src/ja/worker/config.py

```diff
@@ -62,20 +60,10 @@ def to_dict(self) -> Dict[str, object]:
 
     @classmethod
     def from_dict(cls, property_dict: Dict[str, object]) -> "WorkerConfig":
-        if "resource_allocation" in property_dict:
-            ssh_config = SSHConfig.from_dict(
-                cls._get_dict_from_dict(property_dict=property_dict, key="ssh_config", mandatory=True))
-        else:
-            raise ValueError("missing ssh")
-        if "resource_allocation" in property_dict:
-            resource_allocation = ResourceAllocation.from_dict(
-                cls._get_dict_from_dict(property_dict=property_dict, key="resource_allocation", mandatory=False))
-        else:
-            raise ValueError("missing resource allocation")
-
-        if "uid" in property_dict:
-            uid = cls._get_str_from_dict(property_dict, "uid")
-        else:
-            raise ValueError("missing uid")
+        ssh_config = SSHConfig.from_dict(
+            cls._get_dict_from_dict(property_dict=property_dict, key="ssh_config", mandatory=True))
+        resource_allocation = ResourceAllocation.from_dict(
+            cls._get_dict_from_dict(property_dict=property_dict, key="resource_allocation", mandatory=True))
+        uid = cls._get_str_from_dict(property_dict, "uid")
```

Comment:
The uid property is still mandatory because the default value for mandatory is True.
You need to specify mandatory=False.